client/src/components/add-employer-component/addEmployer.jsx

```diff
@@ -35,14 +44,17 @@ function AddEmployer() {
     ],
     jobLeads: [
       {
-        id: 0,
+        id: 1,
```

Comment:
This is causing an error on the job leads card displaying the first job lead as "Job Lead 2"
Further comments will give ways to fix it.